pkg/dom0-ztools/rootfs/etc/init.d/010-eve-cgroup

```diff
@@ -9,7 +9,7 @@ hv=`cat /run/eve-hv-type`
 default_cgroup_cpus_limit=1
 
 default_cgroup_memory_limit=838860800 #800M
-EVESERVICES="sshd edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools newlogd memlogd"
+EVESERVICES="sshd eve-edgeview wwan wlan lisp guacd pillar vtpm watchdog xen-tools newlogd memlogd memory-monitor debug"
```

Comment:
Should we unify cgroup names and avoid the `eve-` prefix which no other service is using?
CC @naiming-zededa

Reply:
Yeah, it would be nicer, but changing only one file is simpler =)